src/chainparams.cpp

```diff
@@ -510,7 +515,7 @@ class CTestNetParams : public CChainParams {
 
         vSeeds.clear();
         // nodes with support for servicebits filtering should be at the top
-        vSeeds.emplace_back("testnet-seed.dashdot.io", true);
+        vSeeds.emplace_back("testnet-seed.dashdot.io"); // Just a static list of stable node(s), only supports x9
```

Comment:
Does anyone know what the `x` and number means? In the BTC PR there are multiple different combinations for seeds.

Reply:
The number is service bits supported by the dns seeder. The idea was that you'd host a seeder on `x<bits>` subdomains to return a list of nodes with only specific bits. Nodes connecting to seeder construct "desired" bits and try to resolve the resulting name. If such subdomain exists - they connect and (presumably) get what they asked for, if not - nodes fallback to connecting to the base domain and get whatever nodes are available.